### PR TITLE
MUZIMA-653: mUzima session timeout valiation

### PR DESCRIPTION
**Pull request by rezita**
I have branched off from an old version, because the I couldn't build the latest version.  The publicly available  muzima-api v1.1.4 jar doesn't contain some newly introduced GPS constants.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -639,6 +639,9 @@
     <string name="warning_switch_server">Switching Server: This will clear all mUzima data on your
         device, including any incomplete forms. Are you sure you want to change this setting?
     </string>
+    <string name="warning_session_timeout">Session Timeout: Could not set session timeout. The
```
Comment (samuelowino) [inline app/src/main/res/values/strings.xml]: Hi @rezita Can you update translations for this string entry in the respective string.xml files

File: app/src/main/java/com/muzima/view/preferences/settings/SettingsPreferenceFragment.java [hunk 2]
```diff
@@ -97,8 +97,20 @@ public boolean onPreferenceChange(final Preference preference, final Object newV
             @Override
             public boolean onPreferenceChange(Preference preference, Object o) {
                 Integer timeOutInMin = Integer.valueOf(o.toString());
-                ((MuzimaApplication) getActivity().getApplication()).resetTimer(timeOutInMin);
-                return true;
+                if (timeOutInMin > 0 && timeOutInMin < 500) {
```
Comment (samuelowino) [inline app/src/main/java/com/muzima/view/preferences/settings/SettingsPreferenceFragment.java]: This looks very clean

Comment (samuelowino) [reply app/src/main/java/com/muzima/view/preferences/settings/SettingsPreferenceFragment.java]: Adjust the PR name to match the ticket name of the ticket you are working on for Example: MUZIMA-111: Refactoring this and that...

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rezita) [reply app/src/main/res/values/strings.xml]: Hello, I have added the translations. I am not sure if they are 100% correct. (I used google translate. My Swahili is not the best. :) )

**Comment by samuelowino** [comment]
Hi @rezita  this is really good work. 
Next step is to create an apk for testing and provide it under the ticket MUZIMA-635:
Write testing instructions on the JIRA for one of the QA team to test.
So far so good Kudos.

Sorry for the late review.
Change ticket status to "TESTING NEEDED".

**Comment by samuelowino** [comment]
Hi!! Just a reminder, you also  need to complete your internship application here... https://forums.muzima.org/t/outreachy-internship-2018-deadline/115

**Comment by samuelowino** [comment]
It would really be helpful to have this as part of your internship pre-tasks..

**Pull request merged**